main_2024.py

```diff
@@ -163,83 +198,89 @@ def main() -> int:
             DETECT_TARGET_OVERRIDE_FULL_PRECISION,
             DETECT_TARGET_SHOW_ANNOTATED,
             DETECT_TARGET_SAVE_PREFIX,
-            video_input_to_detect_target_queue,
-            detect_target_to_data_merge_queue,
-            controller,
         ),
+        [video_input_to_detect_target_queue],
+        [detect_target_to_data_merge_queue],
+        controller,
     )
-
-    flight_interface_manager = worker_manager.WorkerManager()
-    flight_interface_manager.create_workers(
+    flight_interface_worker_args = (
         1,
         flight_interface_worker.flight_interface_worker,
         (
             FLIGHT_INTERFACE_ADDRESS,
             FLIGHT_INTERFACE_TIMEOUT,
             FLIGHT_INTERFACE_WORKER_PERIOD,
-            flight_interface_to_data_merge_queue,
-            controller,
         ),
+        [],
+        [flight_interface_to_data_merge_queue],
+        controller,
     )
-
-    data_merge_manager = worker_manager.WorkerManager()
-    data_merge_manager.create_workers(
+    data_merge_worker_args = (
         1,
         data_merge_worker.data_merge_worker,
-        (
-            DATA_MERGE_TIMEOUT,
+        (DATA_MERGE_TIMEOUT,),
+        [
             detect_target_to_data_merge_queue,
             flight_interface_to_data_merge_queue,
-            data_merge_to_geolocation_queue,
-            controller,
+        ],
+        [data_merge_to_geolocation_queue],
+        controller,
+    )
+    geolocation_worker_args = (
+        1,
+        geolocation_worker.geolocation_worker,
+        (
+            camera_intrinsics,
+            camera_extrinsics,
         ),
+        [data_merge_to_geolocation_queue],
+        [geolocation_to_main_queue],
+        controller,
     )
 
-    result, camera_intrinsics = camera_properties.CameraIntrinsics.create(
-        GEOLOCATION_RESOLUTION_X,
-        GEOLOCATION_RESOLUTION_Y,
-        GEOLOCATION_FOV_X,
-        GEOLOCATION_FOV_Y,
-    )
+    # Create managers
+    worker_managers = []
+
+    result, video_input_manager = worker_manager.WorkerManager.create(*video_input_worker_args)
     if not result:
-        print("Error creating camera intrinsics")
+        frame = inspect.currentframe()
+        main_logger.error("Failed to create manager for Video Input", frame)
         return -1
+    worker_managers.append(video_input_manager)
```

Comment:
Add empty line above this line for separation from the condition block. Do the same for below cases.